config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -0,0 +1,8 @@
+[docker_cli/systemd]
+docker_timeout = 300
```

Comment:
Yes, default is 300. I added that just in case it was changed by user to something shorter in config_custom/defaults.ini. I can remove it if that isn't a good enough reason.